packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -340,7 +340,8 @@ export class AttributesController extends ConsumptionBaseController {
         const shareInfo = LocalAttributeShareInfo.from({
             peer: parsedParams.peer,
             requestReference: parsedParams.requestReference,
-            sourceAttribute: parsedParams.sourceAttributeId
+            sourceAttribute: parsedParams.sourceAttributeId,
+            thirdPartyAddress: parsedParams.thirdPartyAddress
```

Comment:
Yea we decided against it because we want the "user" to know that they are sharing a third party attribute. And that it is not that common to just share relationship attributes.

Reply:
> @sebbi08 Also note that ThirdPartyRelationshipAttributes never can be shared as they are already copies of regular RelationshipAttributes. Only RepositoryAttributes and regular RelationshipAttributes can be shared, whereby the sharing of regular RelationshipAttributes leads to the creation of corresponding copies- the ThirdPartyRelationshipAttributes.

For me it is fine not to validate this here in the AttributesController, since we do so on a higher level in the RequestItemProcessors.  @britsta Are you fine with this, too?

Reply:
Sure! :) It wasn't intended to be a TODO-comment, but more like a sidenote to the sentence:

> And that it is not that common to just share relationship attributes.